### PR TITLE
Add post: Helping users understand and improve the application process

### PR DESCRIPTION
**Pull request by adamsilver**

### CODE REVIEW

File: app/posts/manage-teacher-training-applications/2021-08-10-helping-users-improve-the-application-process.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+---
+title: Helping users understand and improve the application process
+description: First release to help users check how quickly their courses are filling up
+date: 2021-07-26
+tags:
+- MN027
```
Comment (paulrobertlloyd) [inline app/posts/manage-teacher-training-applications/2021-08-10-helping-users-improve-the-application-process.md]: Probably a losing (and unnecessary) battle, but anyway…

```suggestion
  - MN027
```

File: app/posts/manage-teacher-training-applications/2021-08-10-helping-users-improve-the-application-process.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    img: {
+      src: "export-application-data.png"
+    }
+  }, {
+    text: "HESA export now with a new breadcrumb",
+    img: {
+      src: "export-hesa-data.png"
+    }
+  }]
+}) }}
```
Comment (paulrobertlloyd) [inline app/posts/manage-teacher-training-applications/2021-08-10-helping-users-improve-the-application-process.md]: Can now use frontmatter data 😃

File: app/posts/manage-teacher-training-applications/2021-08-10-helping-users-improve-the-application-process.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+## Future considerations
+
+At the moment the application data export and HESA export works across multiple organisations.
+
+In future these pages will be scoped to a single organisation like the status of active applications report.
+
+## Screenshots
+
+Data export page without link to HESA
+Status of active applications report (maybe with and without partner hint text)
```
Comment (paulrobertlloyd) [inline app/posts/manage-teacher-training-applications/2021-08-10-helping-users-improve-the-application-process.md]: Oooooh, you can’t add a description to screenshots, maybe (like related links) you should be able to?

File: app/posts/manage-teacher-training-applications/2021-08-10-helping-users-improve-the-application-process.md (hunk 2 above)
Comment (adamsilver) [reply app/posts/manage-teacher-training-applications/2021-08-10-helping-users-improve-the-application-process.md]: Oh I thought the PR is still open for that.

File: app/posts/manage-teacher-training-applications/2021-08-10-helping-users-improve-the-application-process.md (hunk 3 above)
Comment (adamsilver) [reply app/posts/manage-teacher-training-applications/2021-08-10-helping-users-improve-the-application-process.md]: Nice idea.

**Pull request merged**